### DISCUSSION TITLE
How to backup selected photos?

### DISCUSSION THREAD

**Discussion by dhanadhan** [immich-app/immich#1783] [Q&A]
I am new to Immich, easily the best alternative to Google Photos 👍

Is there a way to individually select and backup those selected photos rather then the entire album/folder? Often times we take some photos to have them temporarily stored for information which I would not like to backup. What's the easiest way to hand pick photos and back up to my server? The dev [mentioned here](https://github.com/immich-app/immich/discussions/1684#discussioncomment-4900986) that we can manually create an album and choose whichever photos we would like to backup. Once an Album is created, it can be backed up from the Immich app. The problem I am facing is that, I don't see the new Album or Folder in the Immich app that I created using Google Photos or Immich app on my Pixel phone. I only see the default 7 Albums/Folders (Recent, Camera, WhatsApp, WhatsApp Images, WhatsApp Video, Screenshots, and Downloads). I have tried clearing Immich app cache and re-logged in with no luck.

I appreciated your input on this or implementation if this feature is to be added in the future release. 

Thanks

**Comment by alextran1502** [comment]
The album you would like to backup will be created from your native Photos viewer; if you are using an Android, that could be GPhotos. So you can create an album name "Backup this" and then from Immich, you would choose that album to be the only one to be uploaded from.

So then, occasionally, you can pick the photo you want to backup to Immich and load it into the "Backup this" album from your native photo app and then run the upload sequence from Immich app.

I hope that makes sense.

**Reply by dhanadhan** [reply]
> The album you would like to backup will be created from your native Photos viewer; if you are using an Android, that could be GPhotos. So you can create an album name "Backup this" and then from Immich, you would choose that album to be the only one to be uploaded from.
> 
> So then, occasionally, you can pick the photo you want to backup to Immich and load it into the "Backup this" album from your native photo app and then run the upload sequence from Immich app.
> 
> I hope that makes sense.

Hi Alex, 

Thank you for the quick reply. 

That is exactly what I tried. Created an album on the Google Photos app, but it doesn't appear on the Immich app for some reason. I have tried logging out of Immich app and cleared cache, but I only see the 7 default folders, not the one I created :(

**Reply by alextran1502** [reply]
Which phone are you using? Maybe Google Photos is not your default Photo viewer? I think Samsung has their own Photo/Gallery app

**Reply by dhanadhan** [reply]
> Which phone are you using? Maybe Google Photos is not your default Photo viewer? I think Samsung has their own Photo/Gallery app

Google Pixel 7. Google Photos is my default photo viewer app.

**Reply by jrasm91** [reply]
Maybe you need at least one photo in it for it to show up. Are there any pictures in the album at the moment?

**Reply by dhanadhan** [reply]
> Maybe you need at least one photo in it for it to show up. Are there any pictures in the album at the moment?

Yes, there are pictures in it. You cannot save an empty album, must have at least one pic.

**Reply by alextran1502** [reply]
If you remove and reinstall Immich, does it show the new album?

**Reply by dhanadhan** [reply]
> If you remove and reinstall Immich, does it show the new album?

No, it does not. Still shows the default 7 folders.

**Reply by alextran1502** [reply]
Hmm how about you create a directory in the Files app?

**Reply by dhanadhan** [reply]
> Hmm how about you create a directory in the Files app?

I am not sure where exactly to create a folder. Tried creating one under Picture directory where Screenshot and WhatsApp folder was, but that didn't show up on Immich either. 

EDIT: Wohoo, I did it. Copied one photo to the folder I created in the Picture directory and it showed up on Immich and Google Photos :)

Thank you for all your help.

**Reply by dhanadhan** [reply]
Another slight problem with the workaround, I have to move/copy photos to this folder rather than just a non-stored copy like how Albums are managed.
